source/quickerNES/core/core.hpp

```diff
@@ -153,7 +153,7 @@ class Core : private Cpu
     if (mapper == nullptr)
     {
       fprintf(stderr, "Could not find mapper for code: %u\n", mapperCode);
-      exit(-1);
+			return "Unsupported mapper";
```

Comment:
Seems to be weird whitespace changes along with tabbing being used where the source files are using spaces for tabs.

Reply:
Must have been because I was editing from inside the submodule.